lib/fog/aws/parsers/compute/describe_images.rb

```diff
@@ -66,9 +66,9 @@ def end_element(name)
               end
             else
               case name
-              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType'
+              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType','creationDate'
```

Comment:
You may wish to separate `creationDate` and parse to a `Time` object.  I cannot find an example of the `creationDate` format so it's hard to determine if it's feasible.